low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -26,8 +26,8 @@
 import os
 import syslog
 import subprocess
-# Add the top level directories
-sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+
+from cortx.sspl.lowlevel.framework.utils.utility import Utility
```

Comment:
Have not changed the Utility import to SimpleProcess as it will be covered in Satish's PR, 
For sspl_setup.py, I have just changed the portion of SetupInit().process adding try catch to it.